spec/support/sign_in_helper.rb

```diff
@@ -1,3 +1,13 @@
+def sign_in(user)
+  if user.is_a?(Admin)
```

Comment:
this is where it gets tricky, are signins cumulative?  e.g, if I sign in as admin, and then voter, does the user object gain the properties of both?  (is that the `user ||=` above?)